### PR TITLE
Updated Spanish translation

### PR DESCRIPTION
**Pull request by EdgarLux**
Translated new lines and some improvements.

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -564,49 +578,61 @@ msgstr ""
...
 
-#: ../build/src/preferences_gen.h:2842
+#: ../build/src/preferences_gen.h:2912
 msgid "defines where scrollbars should be displayed"
 msgstr "define cuándo mostrar las barras de desplazamiento"
 
-#: ../build/src/preferences_gen.h:2850
+#: ../build/src/preferences_gen.h:2920
+msgid "always show panels' scrollbars"
+msgstr "mostrar siempre los páneles de barras de deplazamiento"
```
Comment (agusmba) [inline po/es.po]: The correct translation would be:
"mostrar siempre las barras de desplazamiento de los paneles"

File: po/es.po [hunk 2]
```diff
@@ -564,49 +578,61 @@ msgstr ""
...
+
+#: ../build/src/preferences_gen.h:2932
+msgid ""
+"defines whether the panel scrollbars should be always visible or activated "
+"only depending on the content.  (need a restart)"
+msgstr ""
+"establece si las barras de desplazamiento son permanentes o se activan en "
+"función del contenido"
```
Comment (agusmba) [inline po/es.po]: Maybe add ". (necesita reinicio)" at the end?

File: po/es.po (hunk 1 above)
Comment (EdgarLux) [reply po/es.po]: Thank you! It was corrected.

File: po/es.po (hunk 2 above)
Comment (EdgarLux) [reply po/es.po]: Thank you! It was added.

**Comment by TurboGit** [comment]
Thanks!

**Pull request merged**